configs/openSUSE/dbus-services.toml

```diff
@@ -1231,3 +1231,37 @@ hash = "746f98ec0b4d3ad3fbf71004a55be1f56a74d37a92cb881c7fe266860a1b82ef"
 path = "/usr/share/dbus-1/system.d/org.kde.kinfocenter.dmidecode.conf"
 digester = "xml"
 hash = "20bdf679561f1bc1df39cda8365be3d876a04c4b6d38d9223c6a70ae1f607fc1"
+
+[[FileDigestGroup]]
+package = "ruby3.1-rubygem-d-installer"
+type = "dbus"
+note = "YaST installer backend for e.g. installation over the web browser. IMPORTANT: should only be callable by root or a dedicated user account."
```

Comment:
If this is an instruction or a reminder, I would put it in the imperative mode: "it must be called", "it shall be called", "it is to be called". If this is a hope, e.g. "it should be callable only by root if all goes well", I would re-evaluate this whitelisting altogether :rofl: